example/src/App.tsx

```diff
@@ -38,38 +42,83 @@ export default function App() {
 
   return (
     <SafeAreaView style={styles.container}>
-      <ScrollView>
-        <TestScenario
-          title="Decode SD-JWT"
-          scenario={scenarios.decodeCredentialSdJwt}
-        />
-        <TestScenario
-          title="Verify SD-JWT"
-          scenario={scenarios.verifyCredentialSdJwt}
-        />
-        <TestScenario title="Decode PID" scenario={scenarios.decodePid} />
-        <TestScenario title="Verify PID" scenario={scenarios.verifyPid} />
-        <TestScenario title="Get WIA" scenario={scenarios.getAttestation} />
-        <TestScenario title="Get PID" scenario={scenarios.getPid} />
-        <TestScenario
-          title="Get Credential"
-          scenario={scenarios.getCredential}
-        />
-        <TestScenario
-          title="Get Multiple Credential"
-          scenario={scenarios.getMultipleCredential}
-        />
-        <TestScenario title="Decode QR from RP" scenario={scenarios.decodeQR} />
-        <TestScenario
-          title="Fetch Entity Statement"
-          scenario={scenarios.getEntityStatement}
-        />
-        <TestSameDeviceFlowScenarioWithDeepLink deeplink={deeplink} />
-      </ScrollView>
+      <NavigationContainer>
+        <Tab.Navigator
+          screenOptions={({ route }) => ({
+            // eslint-disable-next-line react/no-unstable-nested-components
+            tabBarIcon: () => {
+              let icon;
+              if (route.name === "PoC") {
+                icon = "🧪";
+              } else if (route.name === "Production") {
+                icon = "🚀";
+              }
+              return <Text>{icon}</Text>;
+            },
+            tabBarActiveTintColor: "blue",
+            tabBarInactiveTintColor: "gray",
+          })}
```

Comment:
Small suggestion here, we could extract this function like this: 
`const bottomBar = (
    route: RouteProp<RootStackParamList, keyof RootStackParamList>
  ) => {
    let icon;
    if (route.name === "PoC") {
      icon = "🧪";
    } else if (route.name === "Production") {
      icon = "🚀";
    }
    return <Text>{icon}</Text>;
  };`
  
  And use it like: 
`tabBarIcon: () => bottomBar(route),`